### ISSUE TITLE
Conversion from Channel[View] to View

### ISSUE THREAD

**Issue by mkotsbak** [widok/widok#81]
Trying a little Wizard framework with this code:

trait WizardPage {
...
  def view(): View
}

val currentWizardStep = Var[WizardPage](InitialStep)

override def view(): View = div(
  currentWizardStep.map(_.view())
)

the last line does not compile as it expects a View, but it has Channel[View]

**Comment by mkotsbak** [comment]
Workaround: put it into a span:

        currentWizardStep.map { wizardStep =>
            span(
                wizardStep.view()
            )
        }

Is there a reason for that? In that case, maybe we could have an implicit convertion.

**Comment by tindzk** [comment]
This should work too:

```
trait WizardPage {
  ...
  def view(): Widget[_]
}
```

**Comment by mkotsbak** [comment]
That worked, thanks. Maybe it still should be a conversion, or it could be possible to make a mini framework for wizards (something like Spring webflow, just better).

**Comment by tindzk** [comment]
Yes, actually I'm thinking about merging ``View`` and ``Widget``. Then we wouldn't need to provide an additional conversion. You could have a look at some of the widgets [here](https://github.com/widok/widok/tree/master/js/src/main/scala/org/widok/bindings/Bootstrap) and create something similar for your wizard.
